contrib/gitian/DOCKRUN.md

```diff
@@ -56,7 +56,7 @@ The dockrun.sh script will do everything to build the binaries. Just specify the
 version to build as its only argument, e.g.
 
 ```bash
-./dockrun.sh v0.17.2.3
+./dockrun.sh v0.17.3.0
```

Comment:
Since this is just an example, does it really need to be updated for every release?

Reply:
I just wanted to bump it while I was in there anyways, it's obviously not required :P